src/Faker/Provider/fr_FR/Company.php

```diff
@@ -166,4 +165,16 @@ protected static function isCatchPhraseValid($catchPhrase)
 
         return true;
     }
+
+    /**
+     * @link http://www.emploi.com/emploi/index-metiers.html
```

Comment:
I suggest you use a public list, like [the list of ROME codes from INSEE](http://www.pole-emploi.fr/candidat/le-code-rome-et-les-fiches-metiers-@/article.jspz?id=60702). It might need filtering as it's very large, but it's a good (and public) datasource.